### ISSUE TITLE
Teleporting issue with enter-claim denied on basic claims

### ISSUE THREAD

**Issue by MJRLegends** [MinecraftPortCentral/GriefPrevention#523]
Hello, 

**Details:**
After days of testing, and talking to the staff over on the discord.

We have come to the conclusion that the teleport system seems to have a bug.

The bug is that when teleporting in to a claim using /home from Nucleus or even modded blocks that have teleporting, the enter-claim module is the issue when that is disabled it works fine.

I was informed to try /minecraft:teleport and that broke too.

**Logs**
FML-Latest_Server.log: https://gist.github.com/MJRLegends/a4909b4c6943b55a6f64b4356392a7eb
GP Debug log:  https://gist.github.com/anonymous/8bfb3a54b8036efe143aa908ca9efe10
Nucleus Diagnostics : https://pastebin.com/0ktr4cvv
Flags Info: https://gyazo.com/e61b682d79c3a1856dccfd04ef9bb48b https://gyazo.com/817ac47784fa8b7b8153e5da4a8fbc10

**Versions**
GriefPrevention 1.10.2-4.0.1.474
SpongeForge 1.10.2-2477-5.2.0-BETA-2698
Luckperms 4.0.6

Feel free to contact Rasgnarok, phit as they were the ones who helped me find this out over on discord.

**Comment by bloodmc** [comment]
@MJRLegends You need to update your GP to latest and provide actual steps to reproduce this issue. Create 2 new claims and retest then provide the exact steps that lead to the bug.

**Comment by MJRLegends** [comment]
@bloodmc I meant to say i was using build 490, i miss copied this information from discord, as i tried many of different versions including the latest. Please talk to Rasgnarok if needed he will confirm that i was indeed using the latest when doing the tests

**Comment by bloodmc** [comment]
@MJRLegends You still need to provide me with the proper information. Create 2 new claims and retest the bug then provide EXACT steps on what you did to reproduce.

**Comment by MJRLegends** [comment]
Making new claims is something that has been already tried by me for this. I even tried wiping all GP data and testing its a reproducible issue

**Comment by bloodmc** [comment]
If you are not going to do what I asked then I will close this issue. I created 2 new claims on a fresh test server and set a home with "enter-claim" set to false and had no issues.

**Comment by MJRLegends** [comment]
At the moment i haven't got a few hours spare to test on a new server

**Comment by phit** [comment]
I tried this a bunch blood, the bug only happens when you are teleporting from far away (target chunk being unloaded).

I made a fresh claim, set enter-claim to false, stood in the center noted down the coords then did `/tp` 1k blocks away and then after a few second `/tp` to the center coords I noted down. Instead of the center it put me on the edge of the claim.

I couldn't really figure out why while debugging, but for some reason enter-claim returned false for the claim owner.
